src/sentry/quotas/base.py

```diff
@@ -503,28 +503,29 @@ def assign_monitor_seat(
         monitor: Monitor,
     ) -> int:
         """
-        Determines if a monitor seat assignment is accepted or rate limited. The Monitor status
-        will be updated from ACTIVE to OK if the seat assignment is accepted.
+        Determines if a monitor seat assignment is accepted or rate limited.
         """
-        from sentry.monitors.models import MonitorStatus
         from sentry.utils.outcomes import Outcome
 
-        monitor.update(status=MonitorStatus.OK)
         return Outcome.ACCEPTED
 
-    def unassign_monitor_seat(
+    def check_accept_monitor_checkin(self, project_id: int, monitor_slug: str):
```

Comment:
What's the difference with the method above?

Reply:
@iker-barriocanal `check_accept_monitor_checkin` is used to see if we already know about and have an accept/drop status for the monitor. Then if we do not have a known status, `assign_monitor_seat` is used to attempt to assign a seat, and it will also get added to the known statuses for the next check-in (the monitors' statuses will be stored in redis).